app/src/androidTest/java/net/skyscanner/backpack/button/BpkButtonTest.kt

```diff
@@ -156,8 +148,8 @@ class BpkButtonTest(flavour: Flavor) : BpkSnapshotTest() {
   }
 
   @Test
+  @Variants(BpkTestVariant.Default)
```

Comment:
I think ideally we should revise our tests and remove thouse variants which we don't need. But not as a part of this PR

Reply:
Can you please create tickets?